modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/LayoutImportController.java

```diff
@@ -591,6 +594,23 @@ else if (ExportImportThreadLocal.isStagingInProcess() &&
 			if (group.isCompany() ^ companySourceGroup) {
 				throw new LARTypeException(LARTypeException.TYPE_COMPANY_GROUP);
 			}
+
+			StagingGroupHelper stagingGroupHelper =
+				StagingGroupHelperUtil.getStagingGroupHelper();
+
+			String groupFriendlyUrl = headerElement.attributeValue(
+				"group-friendly-url");
+
+			boolean companyGroup = stagingGroupHelper.isCompanyGroup(group);
+
+			if (((groupFriendlyUrl == null) && companyGroup) ||
+				(Objects.equals(groupFriendlyUrl, "/export-import") &&
```

Comment:
Should we update this value with the constant `CompanyGroupConstants.FRIENDLY_URL` ?